lib/utils/README.md

```diff
@@ -0,0 +1,15 @@
+# @intellectia/utils
+
+To install dependencies:
+
+```bash
+bun install
```

Comment:
I would like to suggest to use the pnpm commands this is only to remove confusion among the developers. Somewhere it is pnpm command and somewhere it is bun . Can we stuck with only one please . If you want to discuss which one to choose please open the discussion. Thanks